### PR TITLE
fix: typo spelling grammar

### PR DESCRIPTION
**Pull request by slowy07**
hello , i fix some typo in documentation and replace to correct words with reference from [Merriam webster](https://www.merriam-webster.com/), thanks

### CODE REVIEW

File: proposals/2019-07-17-Webhooks.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 ## Introduction
 
-Modern APIs often consist of two-way API traffic, but OpenAPI currently only supports some types of requests. Standard client-to-server API calls are well supported. Server-to-client callbacks are only supported if they are the result of an earlier API call and are documented by nesting under the path of that earlier call. Incoming HTTP reqests ("webhooks") cannot be described in the current version of OpenAPI if they are the result of subscription arranged outside of the scope of the API (e.g. by setting a callback URL in a web interface).
+Modern APIs often consist of two-way API traffic, but OpenAPI currently only supports some types of requests. Standard client-to-server API calls are well supported. Server-to-client callbacks are only supported if they are the result of an earlier API call and are documented by nesting under the path of that earlier call. Incoming HTTP request ("webhooks") cannot be described in the current version of OpenAPI if they are the result of subscription arranged outside of the scope of the API (e.g. by setting a callback URL in a web interface).
```
Comment (servusdei2018) [inline proposals/2019-07-17-Webhooks.md:22]:
```suggestion
Modern APIs often consist of two-way API traffic, but OpenAPI currently only supports some types of requests. Standard client-to-server API calls are well supported. Server-to-client callbacks are only supported if they are the result of an earlier API call and are documented by nesting under the path of that earlier call. Incoming HTTP requests ("webhooks") cannot be described in the current version of OpenAPI if they are the result of subscription arranged outside of the scope of the API (e.g. by setting a callback URL in a web interface).
```
`request` should be in the plural: `requests`

**Review by servusdei2018 — APPROVED** [review]

File: proposals/2019-07-17-Webhooks.md (hunk 1 above)
Comment (slowy07) [reply proposals/2019-07-17-Webhooks.md:22]: okay sir

**Review by servusdei2018 — APPROVED** [review]

**Review by darrelmiller — APPROVED** [review]

**Review by MikeRalphson — APPROVED** [review]

**Comment by MikeRalphson** [comment]
Many thanks for the PR!

**Comment by slowy07** [comment]
thank you sir @MikeRalphson !!

**Pull request merged**